example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -183,6 +183,11 @@
 					97C146ED1CF9000F007C117D = {
 						CreatedOnToolsVersion = 7.3.1;
 						LastSwiftMigration = 0910;
+						SystemCapabilities = {
+							com.apple.BackgroundModes = {
+								enabled = 1;
+							};
+						};
```

Comment:
?

Reply:
Included accidentally from a stashed changes. I'm going to revert it, because it's a part of a different feature.